.github/contributing.rst

```diff
@@ -0,0 +1,13 @@
+Contributing Guideline
+======================
+
+To learn more about the purpose of PEPs and how to go about writing a PEP, please
+start reading at PEP 1 (pep-0001.txt in this repo). Note that PEP 0, the index PEP,
+is now automatically generated, and not committed to the repo.
```

Comment:
I'd like to see a (strong) suggestion added here to first discuss an idea on python-ideas before attempting to write a PEP.